templates/template.rb

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env ruby
-# typed: false
+# typed: ignore
```

Comment:
Sorbet would complain about `NodeKindField#initialize`.
I tried to declare & pass name and comment kwargs explicitly, but it still wasn't happy and only suggested using `T.unsafe`.
It seems weird that Sorbet would fail for something so simple, but anyway it seems like a bug it even complains about this in a `typed: false` file.